### PR TITLE
enable arrow/parquet format in mapserver (full gdal)

### LINKED ISSUE #666: extend gdal with plugin for parquet support
parquet seems an interesting format for mapserver, because it loads only a subset of data in memory

for parquet support the [libgdal-arrow-parquet](https://gdal.org/drivers/vector/parquet.html) should be included

suggestion is to add this lib to the docker build

### PR DESCRIPTION
**Pull request by pvgenuchten**
parquet driver is included in gdal-full + enable GDAL_USE_ARROW

resolves #666

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/osgeo/gdal:ubuntu-small-3.7.3 as gdal
+FROM ghcr.io/osgeo/gdal:ubuntu-full-3.7.3 as gdal
```
Comment (sbrunner) [inline Dockerfile:1]: This image is relay too big to be used by default. If you need that, you should create a separate image!

File: Dockerfile [hunk 2]
```diff
@@ -61,6 +61,7 @@ RUN if test "${WITH_ORACLE}" = "ON"; then \
     -DWITH_RSVG=1 \
     -DUSE_PROJ=1 \
     -DUSE_WFS_SVR=1 \
+    -DGDAL_USE_ARROW=1 \
```
Comment (sbrunner) [inline Dockerfile:64]: What's these build options?
I didn't find any reference in the MapServer sources...

**Review by sbrunner — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (pvgenuchten) [reply Dockerfile:1]: Thanx @SBrunner, 
I agree the full image is too large.
I'm not aware how to build Arrow into GDAL separately (except via conda, is that an option?)
What is your suggestion here?

File: Dockerfile (hunk 2 above)
Comment (pvgenuchten) [reply Dockerfile:64]: Arrow/Parquet support is here:
https://gdal.org/development/building_from_source.html#cmdoption-arg-GDAL_USE_ARROW

**Comment by pvgenuchten** [comment]
I see no easy options to advance the issue, closing for now

**Pull request closed without merging**